src/ProjectTemplates/test/BlazorWasmTemplateTest.cs

```diff
@@ -39,19 +39,19 @@ public async Task BlazorWasmStandaloneTemplate_Works()
             var buildResult = await project.RunDotNetBuildAsync();
             Assert.True(0 == buildResult.ExitCode, ErrorMessages.GetFailedProcessMessage("build", project, buildResult));
 
-            await BuildAndRunTest(project.ProjectName, project);
+            // await BuildAndRunTest(project.ProjectName, project);
```

Comment:
This is effectively the same thing as disabling the test, all of the interesting things happen in BuildAndRunTests and TestBasicNavigation. If we MUST do this for some reason at least file an issue about re-instating it.